doc/man7/OSSL_STORE-winstore.pod

```diff
@@ -0,0 +1,86 @@
+=pod
+
+=head1 NAME
+
+OSSL_STORE-winstore - OpenSSL built in OSSL_STORE for Windows
+
+=head1 DESCRIPTION
+
+The OSSL_STORE implementation for Windows provides access to Windows' system
+C<ROOT> certificate store through URIs, using the URI scheme
+C<org.openssl.winstore>.
+
+=head2 Supported URIs
+
+There is only one supported URI:
+
+  org.openssl.winstore:
+
+No authority (host, etc), no path, no query, no fragment.
+
+=head2 Supported OSSL_STORE_SEARCH operations
+
+=over 4
+
+=item L<OSSL_STORE_SEARCH_by_name(3)>
+
+As a matter of fact, this must be used.  It is not possible to enumerate all
+available certificates in the store.
```

Comment:
Some examples of URIs that you would use to search by name would be helpful

Reply:
Er, the `OSSL_STORE_SEARCH` mechanism isn't URI based.  `OSSL_STORE_SEARCH_by_name()` is passed a `X509_NAME *`.

That being said, an example is not a bad thing.  It will take me some time, though...